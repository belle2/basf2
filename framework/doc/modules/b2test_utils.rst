.. automodule:: b2test_utils
   :members:
