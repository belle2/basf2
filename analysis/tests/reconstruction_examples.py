#!/usr/bin/env python3
# -- analysis/tests/reconstruction_examples.py

##########################################################################
# basf2 (Belle II Analysis Software Framework)                           #
# Author: The Belle II Collaboration                                     #
#                                                                        #
# See git log for contributors and copyright holders.                    #
# This file is licensed under LGPL-3.0, see LICENSE.md.                  #
##########################################################################

import b2test_utils as b2tu
import b2test_utils_analysis as b2tua

#: extend the examples test class to actually run over the reconstruction examples:


class ExamplesTest(b2tua.ExamplesTest):
    def test_reconstruction_examples(self):
        """
        Test supported reconstruction examples.
        """

        self._test_examples_dir('analysis/examples/reconstruction/')


if __name__ == '__main__':
    with b2tu.clean_working_directory():
        b2tua.unittest.main()
