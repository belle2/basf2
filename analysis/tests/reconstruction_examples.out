[INFO] Steering file: analysis/tests/reconstruction_examples.py
TFile: inclusiveDstarReconstruction.root
TTree: Dstzero_slowPizero
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 64
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, 0
TBranch: PDG, 423
TBranch: daughter__bo0__cm__spPDG__bc, 111
TBranch: E, 2.755
TBranch: p, 1.887
TBranch: pt, 1.053
TBranch: useCMSFrame__bop__bc, 1.317
TBranch: useCMSFrame__boE__bc, 2.4
TBranch: matchedMC__bop__bc, 3.029
TBranch: matchedMC__bopt__bc, 0.4565
TBranch: matchedMC__boE__bc, 11
TBranch: matchedMC__bouseCMSFrame__bop__bc__bc, 0.004498
TBranch: matchedMC__boPDG__bc, 3.006e+05
TBranch: px, 0.4963
TBranch: py, -0.9283
TBranch: pz, 1.567
TBranch: M, 2.007
TBranch: InvM, 0.1
TBranch: daughter__bo0__cm__spisSignal__bc, 0
TBranch: daughter__bo0__cm__spE__bc, 0.195
TBranch: daughter__bo0__cm__spp__bc, 0.1673
TBranch: daughter__bo0__cm__spuseCMSFrame__bop__bc__bc, 0.1268
TBranch: daughter__bo0__cm__spmatchedMC__bop__bc__bc, 1.66
TBranch: daughter__bo0__cm__spmatchedMC__boE__bc__bc, 5.534
TBranch: daughter__bo0__cm__spmatchedMC__bouseCMSFrame__bop__bc__bc__bc, 0.3212
TBranch: daughter__bo0__cm__spmatchedMC__boPDG__bc__bc, -521
TBranch: daughter__bo0__cm__spdaughter__bo0__cm__spmatchedMC__boPDG__bc__bc__bc, 22
TBranch: daughter__bo0__cm__spdaughter__bo1__cm__spmatchedMC__boPDG__bc__bc__bc, 22
TBranch: daughter__bo0__cm__spmcMother__boPDG__bc__bc, 3.006e+05
TBranch: daughter__bo0__cm__spmcMother__bop__bc__bc, 3.029
TBranch: daughter__bo0__cm__spmcMother__bouseCMSFrame__bop__bc__bc__bc, 0.004498
TBranch: daughter__bo0__cm__spmcMother__boE__bc__bc, 11
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc, 521
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spp__bc__bc__bc, 1.429
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spE__bc__bc__bc, 5.469
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.3223
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc, -521
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spp__bc__bc__bc, 1.66
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spE__bc__bc__bc, 5.534
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.3212
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc__bc, nan
TTree: Dstcharged_slowPizero
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 60
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, 0
TBranch: PDG, 413
TBranch: daughter__bo0__cm__spPDG__bc, 111
TBranch: E, 2.787
TBranch: p, 1.931
TBranch: pt, 1.077
TBranch: useCMSFrame__bop__bc, 1.353
TBranch: useCMSFrame__boE__bc, 2.423
TBranch: matchedMC__bop__bc, 3.029
TBranch: matchedMC__bopt__bc, 0.4565
TBranch: matchedMC__boE__bc, 11
TBranch: matchedMC__bouseCMSFrame__bop__bc__bc, 0.004498
TBranch: matchedMC__boPDG__bc, 3.006e+05
TBranch: px, 0.5078
TBranch: py, -0.9496
TBranch: pz, 1.603
TBranch: M, 2.01
TBranch: InvM, 0.1
TBranch: daughter__bo0__cm__spisSignal__bc, 0
TBranch: daughter__bo0__cm__spE__bc, 0.195
TBranch: daughter__bo0__cm__spp__bc, 0.1673
TBranch: daughter__bo0__cm__spuseCMSFrame__bop__bc__bc, 0.1268
TBranch: daughter__bo0__cm__spmatchedMC__bop__bc__bc, 1.66
TBranch: daughter__bo0__cm__spmatchedMC__boE__bc__bc, 5.534
TBranch: daughter__bo0__cm__spmatchedMC__bouseCMSFrame__bop__bc__bc__bc, 0.3212
TBranch: daughter__bo0__cm__spmatchedMC__boPDG__bc__bc, -521
TBranch: daughter__bo0__cm__spdaughter__bo0__cm__spmatchedMC__boPDG__bc__bc__bc, 22
TBranch: daughter__bo0__cm__spdaughter__bo1__cm__spmatchedMC__boPDG__bc__bc__bc, 22
TBranch: daughter__bo0__cm__spmcMother__boPDG__bc__bc, 3.006e+05
TBranch: daughter__bo0__cm__spmcMother__bop__bc__bc, 3.029
TBranch: daughter__bo0__cm__spmcMother__bouseCMSFrame__bop__bc__bc__bc, 0.004498
TBranch: daughter__bo0__cm__spmcMother__boE__bc__bc, 11
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc, 521
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spp__bc__bc__bc, 1.429
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spE__bc__bc__bc, 5.469
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.3223
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc, -521
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spp__bc__bc__bc, 1.66
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spE__bc__bc__bc, 5.534
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.3212
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc__bc, nan
TTree: Dstcharged_slowPicharged_negative
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 2
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 1
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, nan
TBranch: PDG, -413
TBranch: daughter__bo0__cm__spPDG__bc, -211
TBranch: E, 2.561
TBranch: p, 1.586
TBranch: pt, 1.586
TBranch: useCMSFrame__bop__bc, 1.701
TBranch: useCMSFrame__boE__bc, 2.634
TBranch: matchedMC__bop__bc, nan
TBranch: matchedMC__bopt__bc, nan
TBranch: matchedMC__boE__bc, nan
TBranch: matchedMC__bouseCMSFrame__bop__bc__bc, nan
TBranch: matchedMC__boPDG__bc, nan
TBranch: px, 0.5479
TBranch: py, 1.488
TBranch: pz, 0.02374
TBranch: M, 2.01
TBranch: InvM, 0.1396
TBranch: daughter__bo0__cm__spisSignal__bc, nan
TBranch: daughter__bo0__cm__spE__bc, 0.1853
TBranch: daughter__bo0__cm__spp__bc, 0.1218
TBranch: daughter__bo0__cm__spuseCMSFrame__bop__bc__bc, 0.1295
TBranch: daughter__bo0__cm__spmatchedMC__bop__bc__bc, nan
TBranch: daughter__bo0__cm__spmatchedMC__boE__bc__bc, nan
TBranch: daughter__bo0__cm__spmatchedMC__bouseCMSFrame__bop__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmatchedMC__boPDG__bc__bc, nan
TBranch: daughter__bo0__cm__spdaughter__bo0__cm__spmatchedMC__boPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spdaughter__bo1__cm__spmatchedMC__boPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__boPDG__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bop__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bouseCMSFrame__bop__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__boE__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spp__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spE__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spuseCMSFrame__bop__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spp__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spE__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spuseCMSFrame__bop__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc__bc, nan
TTree: Dstcharged_slowPicharged
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 2
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 1
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, nan
TBranch: PDG, -413
TBranch: daughter__bo0__cm__spPDG__bc, -211
TBranch: E, 2.561
TBranch: p, 1.586
TBranch: pt, 1.586
TBranch: useCMSFrame__bop__bc, 1.701
TBranch: useCMSFrame__boE__bc, 2.634
TBranch: matchedMC__bop__bc, nan
TBranch: matchedMC__bopt__bc, nan
TBranch: matchedMC__boE__bc, nan
TBranch: matchedMC__bouseCMSFrame__bop__bc__bc, nan
TBranch: matchedMC__boPDG__bc, nan
TBranch: px, 0.5479
TBranch: py, 1.488
TBranch: pz, 0.02374
TBranch: M, 2.01
TBranch: InvM, 0.1396
TBranch: daughter__bo0__cm__spisSignal__bc, nan
TBranch: daughter__bo0__cm__spE__bc, 0.1853
TBranch: daughter__bo0__cm__spp__bc, 0.1218
TBranch: daughter__bo0__cm__spuseCMSFrame__bop__bc__bc, 0.1295
TBranch: daughter__bo0__cm__spmatchedMC__bop__bc__bc, nan
TBranch: daughter__bo0__cm__spmatchedMC__boE__bc__bc, nan
TBranch: daughter__bo0__cm__spmatchedMC__bouseCMSFrame__bop__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmatchedMC__boPDG__bc__bc, nan
TBranch: daughter__bo0__cm__spdaughter__bo0__cm__spmatchedMC__boPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spdaughter__bo1__cm__spmatchedMC__boPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__boPDG__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bop__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bouseCMSFrame__bop__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__boE__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spp__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spE__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spuseCMSFrame__bop__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spp__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spE__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spuseCMSFrame__bop__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc__bc, nan
TTree: Dstzero_slowPizero_MC
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 6
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, 0
TBranch: PDG, 423
TBranch: daughter__bo0__cm__spPDG__bc, 111
TBranch: E, 2.399
TBranch: p, 1.315
TBranch: pt, 0.7486
TBranch: useCMSFrame__bop__bc, 0.8349
TBranch: useCMSFrame__boE__bc, 2.174
TBranch: matchedMC__bop__bc, 1.66
TBranch: matchedMC__bopt__bc, 0.5042
TBranch: matchedMC__boE__bc, 5.534
TBranch: matchedMC__bouseCMSFrame__bop__bc__bc, 0.3212
TBranch: matchedMC__boPDG__bc, -521
TBranch: px, 0.3675
TBranch: py, -0.6522
TBranch: pz, 1.081
TBranch: M, 2.007
TBranch: InvM, 0.135
TBranch: daughter__bo0__cm__spisSignal__bc, 1
TBranch: daughter__bo0__cm__spE__bc, 0.1698
TBranch: daughter__bo0__cm__spp__bc, 0.103
TBranch: daughter__bo0__cm__spuseCMSFrame__bop__bc__bc, 0.06854
TBranch: daughter__bo0__cm__spmatchedMC__bop__bc__bc, 0.103
TBranch: daughter__bo0__cm__spmatchedMC__boE__bc__bc, 0.1698
TBranch: daughter__bo0__cm__spmatchedMC__bouseCMSFrame__bop__bc__bc__bc, 0.06854
TBranch: daughter__bo0__cm__spmatchedMC__boPDG__bc__bc, 111
TBranch: daughter__bo0__cm__spdaughter__bo0__cm__spmatchedMC__boPDG__bc__bc__bc, 22
TBranch: daughter__bo0__cm__spdaughter__bo1__cm__spmatchedMC__boPDG__bc__bc__bc, 22
TBranch: daughter__bo0__cm__spmcMother__boPDG__bc__bc, -521
TBranch: daughter__bo0__cm__spmcMother__bop__bc__bc, 1.66
TBranch: daughter__bo0__cm__spmcMother__bouseCMSFrame__bop__bc__bc__bc, 0.3212
TBranch: daughter__bo0__cm__spmcMother__boE__bc__bc, 5.534
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc, 421
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spp__bc__bc__bc, 0.626
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spE__bc__bc__bc, 1.967
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.1133
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc, -211
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spp__bc__bc__bc, 0.0685
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spE__bc__bc__bc, 0.1555
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.02876
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc__bc, 521
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc__bc, -521
TTree: Dstcharged_slowPizero_MC
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 6
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, 0
TBranch: PDG, 413
TBranch: daughter__bo0__cm__spPDG__bc, 111
TBranch: E, 2.427
TBranch: p, 1.361
TBranch: pt, 0.7746
TBranch: useCMSFrame__bop__bc, 0.8732
TBranch: useCMSFrame__boE__bc, 2.192
TBranch: matchedMC__bop__bc, 1.66
TBranch: matchedMC__bopt__bc, 0.5042
TBranch: matchedMC__boE__bc, 5.534
TBranch: matchedMC__bouseCMSFrame__bop__bc__bc, 0.3212
TBranch: matchedMC__boPDG__bc, -521
TBranch: px, 0.3802
TBranch: py, -0.6748
TBranch: pz, 1.119
TBranch: M, 2.01
TBranch: InvM, 0.135
TBranch: daughter__bo0__cm__spisSignal__bc, 1
TBranch: daughter__bo0__cm__spE__bc, 0.1698
TBranch: daughter__bo0__cm__spp__bc, 0.103
TBranch: daughter__bo0__cm__spuseCMSFrame__bop__bc__bc, 0.06854
TBranch: daughter__bo0__cm__spmatchedMC__bop__bc__bc, 0.103
TBranch: daughter__bo0__cm__spmatchedMC__boE__bc__bc, 0.1698
TBranch: daughter__bo0__cm__spmatchedMC__bouseCMSFrame__bop__bc__bc__bc, 0.06854
TBranch: daughter__bo0__cm__spmatchedMC__boPDG__bc__bc, 111
TBranch: daughter__bo0__cm__spdaughter__bo0__cm__spmatchedMC__boPDG__bc__bc__bc, 22
TBranch: daughter__bo0__cm__spdaughter__bo1__cm__spmatchedMC__boPDG__bc__bc__bc, 22
TBranch: daughter__bo0__cm__spmcMother__boPDG__bc__bc, -521
TBranch: daughter__bo0__cm__spmcMother__bop__bc__bc, 1.66
TBranch: daughter__bo0__cm__spmcMother__bouseCMSFrame__bop__bc__bc__bc, 0.3212
TBranch: daughter__bo0__cm__spmcMother__boE__bc__bc, 5.534
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc, 421
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spp__bc__bc__bc, 0.626
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spE__bc__bc__bc, 1.967
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.1133
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc, -211
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spp__bc__bc__bc, 0.0685
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spE__bc__bc__bc, 0.1555
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.02876
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc__bc, 521
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc__bc, -521
TTree: Dstcharged_slowPicharged_MC
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 5
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, 0
TBranch: PDG, 413
TBranch: daughter__bo0__cm__spPDG__bc, 211
TBranch: E, 2.797
TBranch: p, 1.945
TBranch: pt, 1.545
TBranch: useCMSFrame__bop__bc, 1.613
TBranch: useCMSFrame__boE__bc, 2.577
TBranch: matchedMC__bop__bc, 0.203
TBranch: matchedMC__bopt__bc, 0.1986
TBranch: matchedMC__boE__bc, 0.2463
TBranch: matchedMC__bouseCMSFrame__bop__bc__bc, 0.1985
TBranch: matchedMC__boPDG__bc, 211
TBranch: px, -0.05667
TBranch: py, -1.544
TBranch: pz, 1.181
TBranch: M, 2.01
TBranch: InvM, 0.1396
TBranch: daughter__bo0__cm__spisSignal__bc, 1
TBranch: daughter__bo0__cm__spE__bc, 0.2023
TBranch: daughter__bo0__cm__spp__bc, 0.1465
TBranch: daughter__bo0__cm__spuseCMSFrame__bop__bc__bc, 0.1221
TBranch: daughter__bo0__cm__spmatchedMC__bop__bc__bc, 0.1465
TBranch: daughter__bo0__cm__spmatchedMC__boE__bc__bc, 0.2023
TBranch: daughter__bo0__cm__spmatchedMC__bouseCMSFrame__bop__bc__bc__bc, 0.1221
TBranch: daughter__bo0__cm__spmatchedMC__boPDG__bc__bc, 211
TBranch: daughter__bo0__cm__spdaughter__bo0__cm__spmatchedMC__boPDG__bc__bc__bc, 11
TBranch: daughter__bo0__cm__spdaughter__bo1__cm__spmatchedMC__boPDG__bc__bc__bc, 14
TBranch: daughter__bo0__cm__spmcMother__boPDG__bc__bc, 211
TBranch: daughter__bo0__cm__spmcMother__bop__bc__bc, 0.203
TBranch: daughter__bo0__cm__spmcMother__bouseCMSFrame__bop__bc__bc__bc, 0.1985
TBranch: daughter__bo0__cm__spmcMother__boE__bc__bc, 0.2463
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc, 211
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spp__bc__bc__bc, 0.1465
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spE__bc__bc__bc, 0.2023
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo0__cm__spuseCMSFrame__bop__bc__bc__bc__bc, 0.1221
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spp__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spE__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcDaughter__bo1__cm__spuseCMSFrame__bop__bc__bc__bc__bc, nan
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo0__cm__spPDG__bc__bc__bc__bc, -2212
TBranch: daughter__bo0__cm__spmcMother__bomcMother__bomcDaughter__bo1__cm__spPDG__bc__bc__bc__bc, 211
TFile: inclusiveReconstruction.root
TTree: variables
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 9
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 2468
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: isSignal, 0
TBranch: Mbc, 5.242
TBranch: deltaE, -3.943
TBranch: isUnspecified, 0
TBranch: Xsd_px, 0.3492
TBranch: Xsd_py, -0.34
TBranch: Xsd_pz, 0.3247
TBranch: Xsd_E, 0.9665
TBranch: Xsd_isSignal, 0
TBranch: Xsd_mcErrors, 48
TBranch: Xsd_isUnspecified, 1
TBranch: e_1_px, 0.1423
TBranch: e_1_py, -0.001529
TBranch: e_1_pz, -0.08281
TBranch: e_1_E, 0.1647
TBranch: e_1_isSignal, 0
TBranch: e_1_mcErrors, 132
TBranch: e_1_isUnspecified, 0
TBranch: e_2_px, 0.2234
TBranch: e_2_py, 0.08354
TBranch: e_2_pz, 0.2063
TBranch: e_2_E, 0.3153
TBranch: e_2_isSignal, 1
TBranch: e_2_mcErrors, 0
TBranch: e_2_isUnspecified, 0
TFile: neutralHadron4MomentumCalculation.root
TTree: nbar
TBranch: __experiment__, 1003
TBranch: __run__, 0
TBranch: __event__, 1
TBranch: __production__, 0
TBranch: __candidate__, 0
TBranch: __ncandidates__, 172
TBranch: __eventType__, 
TBranch: __weight__, 1
TBranch: p, 0.01963
TTree: B0
No entry found
TBranch: __experiment__
TBranch: __run__
TBranch: __event__
TBranch: __production__
TBranch: __candidate__
TBranch: __ncandidates__
TBranch: __eventType__
TBranch: __weight__
TBranch: deltaE
TBranch: M
TBranch: daughter__bo2__cmp__bc
TBranch: daughter__bo2__cmclusterE__bc
TFile: reconstructMCDecay.root
TTree: variables
No entry found
TBranch: __experiment__
TBranch: __run__
TBranch: __event__
TBranch: __production__
TBranch: __candidate__
TBranch: __ncandidates__
TBranch: __eventType__
TBranch: __weight__
TBranch: isSignal
TBranch: Mbc
TBranch: deltaE
TBranch: mcErrors
